src/ci/run.sh

```diff
@@ -69,6 +69,11 @@ RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.codegen-units-std=1"
 # space required for CI artifacts.
 RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --dist-compression-formats=xz"
 
+# Enable the `c` feature for compiler_builtins, but only when the `compiler-rt` source is available.
+if [ "$EXTERNAL_LLVM" = "" ]; then
```

Comment:
Will we run into a problem on the dist-various-2 builder, IIRC, that has custom LLVM for a subset of the targets?

Reply:
I'm not sure. I've added it to PR CI temporarily to test.